src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue

```diff
@@ -145,14 +145,16 @@ export default {
 				return p2IsGroup ? -1 : 1
 			}
 
-			let hasSessions1 = !!participant1.sessionIds.length
-			let hasSessions2 = !!participant2.sessionIds.length
+			const hasSessions1 = !!participant1.sessionIds.length
+			const hasSessions2 = !!participant2.sessionIds.length
+			/*
```

Comment:
remove commented out block or are you still testing ?

Reply:
Keeping it as a comment similar to https://github.com/nextcloud/spreed/pull/4253/files#diff-4eb65f9626b81feabc22bb4e044ba305cbe75667d29909103aa935e97c081dcb so it's easy to see that it is missing intentionally

Reply:
in that case I'd suggest adding a text comment as well because the next developer will wonder if this was forgotten

tbh I don't really understand why commenting this out fixes the issue, so maybe the comment could also add a hint about that

Reply:
the problem is sorting accepted the user_status but the "show online/offline" ignored it 🤷

Reply:
Added a comment in both places